tests/main/snap-confine-tmp-mount/task.yaml

```diff
@@ -25,6 +25,9 @@ debug: |
 
 execute: |
     SNAP_CONFINE=$(os.paths libexec-dir)/snapd/snap-confine
+    if tests.info is-snapd-pkg-repo; then
+        SNAP_CONFINE="/snap/snapd/current$SNAP_CONFINE"
```

Comment:
This should use snapd snap mount dir as this test executes in various distributions.

Reply:
maybe needs a TODO or a comment, that the is-snapd-pkg-repo only works for deb at the moment, hence the distro where this scenario is executed all use `/snap`. But yeah, it should use mount dir if we want to avoid coming back to the test later